### PR TITLE
update nicobar page with tribe info

### PR DESCRIPTION
**Pull request by rejishashi**
- [x] Added the Tribe info below the tourist locations

- [x] Added the below tribe info - Negrito, Great Andamanese, Onge, Jarawa

### CODE REVIEW

File: nicobar_islands.html [hunk 1]
```diff
@@ -74,6 +74,25 @@
         </div>
 
     </div>
+  
+    <div>  
+      <br/><br/>
+        <strong>The Tribes of Andaman </strong> <br/>
```
Comment (rejimon) [inline nicobar_islands.html:80]: The content is good to being with...

File: nicobar_islands.html [hunk 2]
```diff
@@ -74,6 +74,25 @@
...
+
+        <p>Great Andamanese: Population 43. The tribe that has suffered most from contact with outsiders: 99% have been wiped out since the British first colonized the islands. Before the tsunami they lived in a government settlement and were dependent on government aid. They have just been moved to the islands' capital, Port Blair, as their village was badly damaged in the tsunami. </p>
+
+        <p>Onge: Population 100. Their forest home has been plundered by poachers and loggers. They were settled by the Indian administration and are dependent on food handouts. The 73 Onge living at Dugong Creek fled to high ground when they saw the sea level fall, and so survived. </p>
+
+        <p>Jarawa: Population about 270. Have only had peaceful contact with outsiders for six years. They live on the west coast of South and Middle Andaman, and are believed to have survived the tsunami. They are still completely independent and live entirely by hunting, gathering and fishing. The main threat to their existence comes from the highway running through their territory: the Indian government was ordered to close this by the Supreme Court in 2002, but it has ignored the order. </p>
+
```
Comment (rejimon) [inline nicobar_islands.html:98]: Please add information on these tribes as well Sentinelese, Shompen and Nicobarese.
Also seperate the title of each tribe with the text in the line below.

File: nicobar_islands.html [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
        </p>
        <br/>
-       Tourist Attractions at the Nicobar Islands
+       <strong>Tourist Attractions at the Nicobar Islands</strong>
```
Comment (rejimon) [inline nicobar_islands.html:36]: Looks good now

**Review by rejimon — CHANGES_REQUESTED** [review]
Please add information on these tribes as well Sentinelese, Shompen and Nicobarese
Also seperate the title of each tribe with the text in the line below

File: nicobar_islands.html (hunk 2 above)
Comment (rejishashi) [reply nicobar_islands.html:98]: added the new tribe info

**Review by rejimon — APPROVED** [review]
Looks add good, let push these across.

**Pull request merged**